containerfixture/containerfixture.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "test-pod"
+spec:
+  containers:
+  - args:
+    - start-service
+    env:
+    - name: SPLUNK_START_ARGS
+      value: --accept-license
+    - name: SPLUNK_PASSWORD
+      value: Password
+    image: splunk
```

Comment:
the file name suggests this is meant to start any container - but the image is fixed to the splunk image...

The ports section may also need to be different for different containers.

Reply:
Not sure how can we generalize this file. it's not only the image name, but the args, ports, and env vars. we might as well just have a different yaml for every service (or have one for all of them, but containing multiple definitions, not dynamic ones)

Reply:
Using a template can help here - each fixture can have its own template

Reply:
I've implemented templates into it, but I don't see how replacing complete sections is readable or better than having separate files...

Reply:
I did say those were mutually exclusive

Reply:
I can implement a separate template for every fixture; but since we cannot embed dynamically, it's either embedding each file separately (and then somehow using the specific one we need?) or embedding a filesystem, which I'm not sure is desirable

Reply:
Embed to the image-specific fixture and pass as a parameter to the generic "base" fixture.